src/main/resources/db/changelog/initial-inserts.yaml

```diff
@@ -0,0 +1,162 @@
+users:
```

Comment:
Let's not add/change the liquibase scripts in that PR(especially changing the applied one's) Seems odd to do such things in the readme PR, do a separate one